MediaBrowser.LocalMetadata/Parsers/BaseItemXmlParser.cs

```diff
@@ -449,7 +449,7 @@ protected virtual void FetchDataFromXmlNode(XmlReader reader, MetadataResult<T>
 
                     if (!string.IsNullOrWhiteSpace(val))
```

Comment:
While it might seem this handles blanks, it doesn't handle _trailing_ or _leading_ blanks, just _only_ blanks.